File: Microservices.Answers/Entities/Models/Answer.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Microservices.Answers.Entities.Models
{
    public class Answer
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public Guid Id { get; set; }

        [Required]
        public string Text { get; set; }

        [Required]
        public DateTime PublishedAt { get; set; }

        public DateTime LastEditedAt { get; set; }

        public string AuthorId { get; set; }

        //TODO: add attribute
        public Guid PostId { get; set; }

        public Guid AnswerId { get; set; }

        public int AnswersCount { get; set; }
    }
}

File: WebClient/ClientApp/src/app/models/answer.ts
import { Post } from "./post";
import { User } from "./user";

export class Answer {
    
    constructor(
        public id: string,
        public text: string,
        public publishedAt: string,
        public answersCount: number,
        public post: Post,
        public author: User
      ) {  }
}